solvers/lasso_jl.jl

```diff
@@ -0,0 +1,36 @@
+using Distributions
+using GLM
+using Lasso
+using LinearAlgebra
+using PyCall
+using SparseArrays
+
+function scipyCSC_to_julia(A)
+    m, n = A.shape
+    colPtr = Int[i + 1 for i in PyArray(A."indptr")]
+    rowVal = Int[i + 1 for i in PyArray(A."indices")]
+    nzVal = Vector{Float64}(PyArray(A."data"))
+    B = SparseMatrixCSC{Float64,Int}(m, n, colPtr, rowVal, nzVal)
+
+    return B
+end
+
+function solve_lasso(
+    X, y::Vector{Float64}, lambda::Vector{Float64}, fit_intercept::Bool, tol::Float64
+)
+    lasso_fit = fit(
+        LassoPath,
+        X,
+        y;
+        λ=lambda,
+        standardize=false,
+        intercept=fit_intercept,
+        randomize=false,
+        stopearly=false,
+        maxncoef=max(size(X, 1), size(X, 2)) * 100,
```

Comment:
can you comment on what this is?

Reply:
randomize shuffles indices before cd loops (sometimes or always, I don't know), stopearly is for stopping the path (change in deviance, saturation), maxncoef is also related to stopping the path early (maximum number of nonzero coefficients). I ran into a bug in the package when these were at their defaults, but I think only the maxncoef setting is actually necessary to work around it. I'll experiment a bit. randomize should really be true I think, but the others do not really matter.